src/plugins/discover/public/application/components/default_discover_table/table_row.tsx

```diff
@@ -107,18 +107,33 @@ const TableRowUI = ({
         const sanitizedCellValue = dompurify.sanitize(formattedValue);
 
         if (!fieldInfo?.filterable) {
-          return (
-            <td
-              key={colName}
-              data-test-subj="docTableField"
-              className="osdDocTableCell eui-textBreakAll eui-textBreakWord"
-            >
-              <div className="truncate-by-height">
-                {/* eslint-disable-next-line react/no-danger */}
-                <span dangerouslySetInnerHTML={{ __html: sanitizedCellValue }} />
-              </div>
-            </td>
-          );
+          if (colName === '@timestamp') {
```

Comment:
Duplicating this big block just to add a class is detrimental to the maintainability of this piece of code. Please use `classNames` to conditionally add `eui-textBreakAll`.